contrib/BBEditTextWrangler-julia.plist

```diff
@@ -843,10 +843,9 @@
               <string>rationalize</string>
               <string>read</string>
               <string>read!</string>
-              <string>readall</string>
+              <string>readstring</string>
```

Comment:
... and will presumably be re-sorted next time the file is generated [`<!-- these generated from names(Base) -->`](https://github.com/samoconnor/julia/blob/issue_14608/contrib/BBEditTextWrangler-julia.plist#L77).
For now I think its better for the diff to be minimal so it is clear that a was renamed to b.